WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -391,15 +391,22 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
     }
 
     func updateComments(_ comments: [Comment], totalComments: Int) {
+        guard let post = post else {
+            DDLogError("Missing post when updating Reader post detail comments.")
+            return
+        }
+
         // Set the delegate here so the table isn't shown until fetching is complete.
         commentsTableView.delegate = commentsTableViewDelegate
         commentsTableView.dataSource = commentsTableViewDelegate
 
         commentsTableViewDelegate.updateWith(comments: comments,
                                              totalComments: totalComments,
-                                             commentsEnabled: post?.commentsOpen ?? false,
-                                             followingEnabled: post?.canSubscribeComments ?? false,
+                                             commentsEnabled: post.commentsOpen,
+                                             followingEnabled: post.canSubscribeComments,
+                                             isSubscribedComments: post.isSubscribedComments,
```

Comment:
I think it may be better just to pass the `post` here instead of individual properties, but I'll look at that later when I add the Follow functionality.